python-avocado.spec

```diff
@@ -51,6 +51,7 @@ BuildRequires: python-requests
 BuildRequires: python-resultsdb_api
 BuildRequires: python-setuptools
 BuildRequires: python-sphinx
+BuildRequires: python-six
```

Comment:
ok, but the commit is out of order.

Reply:
GH sorts it by date.  If you look at the commits themselves on Git, they're on the right order.  I can fix that with a rebase `--ignore-date`, in a v2.

Reply:
oh, my bad